email/registration.html

```diff
@@ -0,0 +1,26 @@
+<div>
+  <p>All Glories to Sri Sri Guru and Gauranga!</p>
+  <p>Hare Krsna. Dandavat pranams dear {{params.user.name}}.</p>
+  <br>
+
+  <p>Thank you for choosing to be a part of Srila Gurudeva’s Audio Seva Team!</p>
+  <p>We will soon email you further details about your seva assignment.</p>
+  <br>
+
+  <p>Your Servants,</p>
+  <p>Audio Seva Team.</p>
+  <br>
+</div>
+
+<hr>
+
+<div>
+  <p><b>Name:</b> {{params.user.name}}</p>
+  <p><b>Country:</b> {{params.user.location}}</p>
+  <p><b>Email:</b> {{params.user.emailAddress}}</p>
+  <p><b>Contact number:</b> {{params.user.phoneNumber}}{{if(params.user.isAvailableOnWhatsApp; ", available on WhatsApp")}}</p>
+  <p><b>Languages:</b> {% for language in params.user.languages %} {{language}} {{if(!forloop.Last; ", ")}} {% endfor %}</p>
```

Comment:
Isn't there a `join` function in the template language? Just wondering.

Reply:
No, but there's a `join` filter.

Reply:
Soo have you used it now?